src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/IAnalyzeResultWriter.cs

```diff
@@ -10,5 +10,7 @@ namespace Microsoft.DotNet.UpgradeAssistant.Analysis
     public interface IAnalyzeResultWriter
     {
         Task WriteAsync(IAsyncEnumerable<AnalyzeResultDefinition> results, string? format, CancellationToken token);
+
+        string GetFormat();
```

Comment:
```suggestion
        string Format { get; }
```

Properties feel more natural for this kind of API where there's no work done and it's just getting a value.

Reply:
Resolved